db/migrate/20200409043946_create_classified_listing_categories.rb

```diff
@@ -0,0 +1,11 @@
+class CreateClassifiedListingCategories < ActiveRecord::Migration[5.2]
+  def change
+    create_table :classified_listing_categories do |t|
+      t.string :name, null: false, unique: true
```

Comment:
I don't think `unique: true` is working, see the schema.rb. 

I think you need `add_index(:classified_listings_categories, :name, unique: true)` after the `create_table`

Reply:
You're absolutely right, the migration DSL is my eternal nemesis, I like plain SQL so much more. Will fix in a sec.